cartridges/int_twilio/cartridge/controllers/ProductSubscription.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/**
+ * @namespace ProductSubscription
+ */
+
+var server = require('server');
+var consentTracking = require('*/cartridge/scripts/middleware/consentTracking');
+var csrfProtection = require("*/cartridge/scripts/middleware/csrf");
+
+server.get('Show', server.middleware.https, consentTracking.consent, csrfProtection.generateToken, function (req, res, next) {
```

Comment:
You can add a middleware for remote include only as this won't be loaded without remote include